### ISSUE TITLE
Incomplete autocorrect for comment directive pair

### ISSUE THREAD

**Issue by ryym** [bbatsov/rubocop#5689]
Currently RuboCop provides these cops:

- [`Lint/UnneededCopDisableDirective`](https://github.com/bbatsov/rubocop/blob/master/manual/cops_lint.md#lintunneededcopdisabledirective)
- [`Lint/UnneededCopEnableDirective`](https://github.com/bbatsov/rubocop/blob/master/manual/cops_lint.md#lintunneededcopenabledirective)

They are so useful and support the auto correction. However, it seems that their auto correction cannot perform simultaneously.

Example code (sample.rb):

```ruby
# frozen_string_literal: true

# rubocop:disable Metrics/MethodLength
class Empty; end
# rubocop:enable Metrics/MethodLength
```

--------

## Expected behavior

Running `rubocop -a sample.rb` removes both of the unnecessary comment directives.

```ruby
# frozen_string_literal: true

class Empty; end
```

## Actual behavior

Running `rubocop -a sample.rb` removes the `rubocop:disable` comment only.

```ruby
# frozen_string_literal: true
class Empty; end
# rubocop:enable Metrics/MethodLength
```

And running `rubocop -a sample.rb` again removes the `rubocop:enable` comment as well.

```ruby
# frozen_string_literal: true

class Empty; end
```

## Steps to reproduce the problem

Run `rubocop -a sample.rb` with an empty `rubocop.yml` config file.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.53.0 (using Parser 2.5.0.3, running on ruby 2.4.2 x86_64-darwin16)
```

related issue: https://github.com/bbatsov/rubocop/issues/5356

----

Personally, I don't think this must be fixed (because it may be a error-prone process to detect and remove disable/enable comment pairs correctly). But I also think there should be an issue about this behavior so I just reported it.

**Comment by ryym** [comment]
Sorry, I found this was the duplicate of https://github.com/bbatsov/rubocop/issues/4528.
But now RuboCop has the new `Lint/UnneededCopEnableDirective` cop that did not exist at #4528, so the situation is a little different from that time.
